src/cellviz.cpp

```
@@ -32,6 +33,37 @@
+    gtk_combo_box_text_append_text(GTK_COMBO_BOX_TEXT(mean_median_toggle), "Median");
+    gtk_combo_box_set_active(GTK_COMBO_BOX(mean_median_toggle), 0);
+
+    // add all
+    GtkWidget *vbox = gtk_box_new(GTK_ORIENTATION_VERTICAL, 5);
+    gtk_container_add(GTK_CONTAINER(windowg), vbox);
+    gtk_box_pack_start(GTK_BOX(vbox), label, TRUE, TRUE, 0);
+    gtk_box_pack_start(GTK_BOX(vbox), mean_median_toggle, TRUE, TRUE, 0);
+
+    // add button to close the window
+    GtkWidget *button = gtk_button_new_with_label("Run Simulation");
+    g_signal_connect_swapped(button, "clicked", G_CALLBACK(gtk_widget_destroy), windowg);
```

cppcheck (2.10-2)

[performance] Variable 'species' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)